api/RTCPeerConnection.json

```diff
@@ -110,33 +104,27 @@
           "support": {
             "chrome": [
               {
-                "version_added": "55",
-                "notes": "Before Chrome 63 the default value for the <code>RTCConfiguration.rtcpMuxPolicy</code> parameter was <code>&quot;negotiate&quot;</code>"
+                "version_added": "56",
+                "notes": "In Chrome 63, the default value for the <code>configuration.rtcpMuxPolicy</code> parameter changed from <code>&quot;negotiate&quot;</code> to <code>&quot;required&quot;</code>."
```

Comment:
Can you update the notes in a separate PR? I'd like to ask @ddbeck to review them instead of me.